lib/sprockets/sassc_processor.rb

```diff
@@ -0,0 +1,77 @@
+require 'sprockets/sassc_importer'
+require 'sprockets/sass_processor'
+
+module Sprockets
+  class SasscProcessor < SassProcessor
+    def initialize(options = {}, &block)
+      @cache_version = options[:cache_version]
+      @cache_key = "#{self.class.name}:#{VERSION}:#{Autoload::SassC::VERSION}:#{@cache_version}".freeze
+      @importer_class = options[:importer] || SasscImporter
+      @functions = Module.new do
+        include SassProcessor::Functions
+        include Functions
+        include options[:functions] if options[:functions]
+        class_eval(&block) if block_given?
```

Comment:
Actually, reading over this again (albeit now later and through more tired eyes), I don’t see an alternative to doing it like this since we need a handle on that otherwise anonymous block anyway.